packages/app/screens/trip/TripDetails.tsx

```diff
@@ -1,52 +1,45 @@
-import React, { useEffect, useMemo } from 'react';
-import { DetailsHeader } from '../../components/details/header';
+import React, { useEffect } from 'react';
 import { createParam } from 'solito';
-import { TableContainer } from '../../components/pack_table/Table';
-import { selectPackById } from '../../store/packsStore';
 import { useSelector, useDispatch } from 'react-redux';
 import { View } from 'react-native';
 import { RText } from '@packrat/ui';
 import { DetailsComponent } from '../../components/details';
 import { Platform, StyleSheet } from 'react-native';
-import { theme } from '../../theme';
 import { CLIENT_URL } from '@env';
-import ScoreContainer from '../../components/ScoreContainer';
-import WeatherCard from '../../components/weather/WeatherCard';
-import TripCard from '../../components/TripCard';
-import { FontAwesome5 } from '@expo/vector-icons';
-import { convertPhotonGeoJsonToShape } from '../../utils/mapFunctions';
 import useTheme from '../../hooks/useTheme';
 import useCustomStyles from 'app/hooks/useCustomStyles';
-import { useFetchSingleTrip, useTripWeather } from 'app/hooks/singletrips';
-import { RootState } from 'store/store';
+import { useFetchSingleTrip } from 'app/hooks/singletrips';
+import useWeather from './useWeather';
+import {
+  TableContainerComponent,
+  WeatherCardComponent,
+  TripCardComponent,
+  ScoreContainerComponent,
+  loadStyles,
+} from './TripDetailsComponents';
 
 const { useParam } = createParam();
 
 export function TripDetails() {
+  const { weatherObject, weatherWeek } = useWeather();
   const { enableDarkMode, enableLightMode, isDark, isLight, currentTheme } =
     useTheme();
   const styles = useCustomStyles(loadStyles);
-  console.log('trip detail');
-  const dispatch = useDispatch();
   const [tripId] = useParam('tripId');
 
   // console.log("🚀 ~ file: TripDetails.js:34 ~ TripDetails ~ tripId:", tripId)
   const { data, isLoading, error, refetch, isOwner, isError } =
     useFetchSingleTrip(tripId);
 
-  const { weatherObject, weatherWeek } = useTripWeather(data);
```

Comment:
why remove this ?

Reply:
> why remove this ?

Meant to manage it inside useWeather hook. to create its own hook, instead of inside app/hooks/signleTrips hook component. now I think is of no use, so I revoked.